src/main/java/hudson/remoting/Launcher.java

```diff
@@ -227,6 +245,12 @@ public void run() throws Exception {
             if (this.noReconnect) {
                 jnlpArgs.add("-noreconnect");
             }
+            if (candidateCertificates != null && !candidateCertificates.isEmpty()) {
+                for (String c: candidateCertificates) {
+                    jnlpArgs.add("-cert");
+                    jnlpArgs.add(c);
```

Comment:
```
        Main m = new Main();
        CmdLineParser p = new CmdLineParser(m);
        p.parseArgument("-cert", "foo", "-cert", "fbar", "-cert=x,y", "-cert", "z,w");
        System.out.println(m.candidateCertificates);
```

prints

```
[foo, fbar, x,y, z,w]
```

So it handles multiple certificates. it does not do comma separated